docs/filters.md

```diff
@@ -244,9 +244,17 @@ Matches issues of the given repository. If omitted, will match issues of the def
 
 ### sort
 
-*Expects a comma-separated list of sorting keys. For example, `repo, ~updated, -comments`. Any negation operator can be used to invert them. Keys can be any of `comments`, `repo`, `updated`, `date` (an alias for `updated`), `id`, `assignee` or a label group. Label groups can be disambiguated by appending a `.`.*
-
-Sorts a repository by the list of keys, going from left to right. Negated keys will reverse the ordering that they describe.
+Sorts a repository by the list of criteria, going from left to right. Negated criteria will reverse the ordering that they describe.
+
+*Expects a comma-separated list of sorting criteria. For example, `repo, ~updated, -comments`. Here are the available sort criteria:*
```

Comment:
Looks like in all other places the section starts with the italic *Expects ...* sentence. You may want to follow the same. Other than that, can merge.